lib/Lec6-BinaryHeap/binaryHeap.js

```diff
@@ -0,0 +1,41 @@
+let array_length;
+let heap_root = (input, i) => {
+    let left = 2 * i + 1;
+    let right = 2 * i + 2;
+    let maxValue = i;
+
+    if (left < array_length && input[left] > input[maxValue]) {
+        maxValue = left;
+    }
+    if (right < array_length && input[right] > input[maxValue]) {
+        maxValue = right;
+    }
+
+    if (maxValue != i) {
+        swap(input, i, maxValue);
+        heap_root(input, maxValue);
+    }
+}
+let swap = (input, index_One, index_Two) => {
+    var temp = input[index_One];
```

Comment:
Combination of `camel-case` and `snake-case`? It is really hard to read. You should accord to one code style.